src/main/java/no/sikt/nva/pubchannels/handler/model/PublisherDto.java

```diff
@@ -20,14 +23,23 @@ public record PublisherDto(
     String discontinued,
     String year,
     ScientificValueReviewNotice reviewNotice)
-    implements JsonSerializable {
+    implements PublicationChannelDto, JsonSerializable {
 
   public static final String TYPE = "Publisher";
 
   public static PublisherDto create(
       URI selfUriBase, ThirdPartyPublisher publisher, String requestedYear) {
-    var year = Optional.ofNullable(publisher.getYear()).orElse(requestedYear);
-    var id = UriWrapper.fromUri(selfUriBase).addChild(publisher.identifier(), year).getUri();
+    var year =
+        publisher
+            .getYear()
+            .or(() -> Optional.ofNullable(requestedYear))
+            .orElse(String.valueOf(LocalDate.now().getYear()));
```

Comment:
Is it intended/necessary to "fake" the year if missing from both request and response?
I tried replacing this with `null` and all the tests still pass, so maybe we need extra testing in that case:

```
var year = publisher.getYear().or(() -> Optional.ofNullable(requestedYear)).orElse(null);
```

(same applies to `SerialPublicationDto`)